### ISSUE TITLE
AtomEnvironment.showSaveDialogSync is deprecated.

### ISSUE THREAD

**Issue by caseywatts** [sverweij/atom-graphviz-preview-plus#33]
(atom automated this issue for us - so cool)

atom.showSaveDialogSync is deprecated and will be removed soon.

      Please, implement ::saveAs and ::getSaveDialogOptions instead for pane items
      or use Pane::saveItemAs for programmatic saving.

```
AtomEnvironment.showSaveDialogSync (<embedded>:1434:11)
GraphVizPreviewView.saveAs (/Users/cwatts/.atom/packages/graphviz-preview-plus/lib/graphviz-preview-plus-view.js:568:39)
HTMLDivElement.graphvizPreviewPlusSaveAsPng (/Users/cwatts/.atom/packages/graphviz-preview-plus/lib/graphviz-preview-plus-view.js:280:28)
CommandRegistry.handleCommandEvent (<embedded>:23670:49)
CommandRegistry.dispatch (<embedded>:23559:23)
AtomEnvironment.dispatchContextMenuCommand (<embedded>:1587:25)
```

**Comment by sverweij** [comment]
hi @caseywatts thanks for reporting this. I bumped into it as well and opened #31 for it - but other engagements have kept me from implementing it. 

I'll keep this issue open though, for now, so atom's automated reporter will report it as reported

**Comment by sverweij** [comment]
@caseywatts I've published atom-graphviz-preview-plus v1.5.4 a few minutes ago, which contains a fix for this issue.
